pkg/operator/ceph/admission/validate.go

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
How about renaming this file to admission.go and moving it up a directory? Let's think of this validation as part of the operator instead of a separate admission package.

Reply:
I've moved it up as suggested.